test/javascripts/acceptance/pricing-table-test.js

```diff
@@ -0,0 +1,50 @@
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import { visit } from "@ember/test-helpers";
+import { test } from "qunit";
+
+acceptance("Discourse Subscriptions", function (needs) {
+  needs.user();
+  needs.settings({
+    discourse_subscriptions_pricing_table: JSON.stringify({
+      pricingTableId: "pricingTableId",
+      publishableKey: "publishableKey",
+    }),
+  });
+
+  test("pricing table element includes email", async function (assert) {
+    await visit("/subscriptions");
+
+    assert.equal(
+      document.querySelector("stripe-pricing-table").outerHTML,
```

Comment:
You can use qunit-dom for testing DOM elements instead.
Here are a few examples:

```javascript
  let pricingTableElement = document.querySelector("stripe-pricing-table");
  
  assert.dom(pricingTableElement).exists('Should find a stripe-pricing-table');
  assert.dom(pricingTableElement).hasAttribute('pricing-table-id', 'pricingTableId', 'Should have a pricing-table-id');
  assert.dom(pricingTableElement).hasAttribute('publishable-key', 'publishableKey', 'Should have a publishable-key');
  assert.dom(pricingTableElement).hasAttribute('customer-email', 'eviltrout@example.com', 'Should have a customer-email');

```